web/template/layout/mlpa-progress-indicator.gohtml

```diff
@@ -0,0 +1,28 @@
+{{ define "mlpa-progress-indicator" }}
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <div class="govuk-details__summary-text app-progress-indicator-summary">
+
+        {{ $indicatorProgress := "complete" }}
+        {{ if eq .Status "IN_PROGRESS" }}
+          {{ $indicatorProgress = "in-progress" }}
+        {{ else if eq .Status "NOT_STARTED" }}
+          {{ $indicatorProgress = "not-started" }}
+        {{ end }}
+
+        <svg xmlns="http://www.w3.org/2000/svg" class="app-progress-indicator-icon" viewBox="0 0 16 16" data-progress-indicator="{{ $indicatorProgress }}">
+          <ellipse data-svg-elt="background" cx="8" cy="8" rx="7" ry="7"></ellipse>
+          <path data-svg-elt="border" d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14m0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16"/>
+          <path data-svg-elt="tick" d="m10.97 4.97-.02.022-3.473 4.425-2.093-2.094a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-1.071-1.05"/>
+        </svg>
+
+        <div>
+          <h3 class="govuk-!-display-inline">{{ progressIndicatorContext .Indicator }}</h3>
```

Comment:
The progress status is currently only indicated visually, so needs some text for screenreaders. Should be sufficient to append it here with an SR only class:
```suggestion
          <h3 class="govuk-!-display-inline">{{ progressIndicatorContext .Indicator }} <span class="govuk-visually-hidden">({{ translationFunction .Status }})</h3>
```